repository/pom.xml

```diff
@@ -63,5 +63,15 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.1</version>
+        </dependency>
```

Comment:
Please specify the version numbers in the main `pom.xml` dependencyManagement section

Reply:
I have moved the version numbers to main pom.xml's dependencyManagement section and updated the pull request.